### PR TITLE
STACK-161, CGNv6 Fucntional Playbooks

### PR DESCRIPTION
**Pull request by swapkashid**
a10_cgnv6_dns64
a10_cgnv6_logging
a10_cgnv6_one_to_one_pool
a10_cgnv6_scaleout
a10_cgnv6_server
a10_cgnv6_service_group
a10_cgnv6_sixrd

### CODE REVIEW

File: examples/a10_cgnv6_dns64/a10_cgnv6_dns64.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      a10_host: "{{ a10_host }}" 
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      state: "present"
+      partition: "MyPartition"
+      get_type: "single"
+      sampling_enable:
+      - counters1: "all"
```
Comment (hthompson6) [inline examples/a10_cgnv6_dns64/a10_cgnv6_dns64.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_logging/a10_cgnv6_logging.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      state: "present"
+      nat_quota_exceeded:
+        level: "warning"
+      sampling_enable:
+      - counters1: "all"
```
Comment (hthompson6) [inline examples/a10_cgnv6_logging/a10_cgnv6_logging.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_one_to_one_pool/a10_cgnv6_one_to_one_pool_group.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    a10_cgnv6_one_to_one_pool_group:
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      state: "present"
+      get_type: "single"
+      member_list:
+      - pool_name: "MyPool"
```
Comment (hthompson6) [inline examples/a10_cgnv6_one_to_one_pool/a10_cgnv6_one_to_one_pool_group.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_server/a10_cgnv6_server.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    a10_cgnv6_server.py:
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      state: "present"
+      health_check_disable: "false"
+      port_list:
+      - health_check_disable: "true"
```
Comment (hthompson6) [inline examples/a10_cgnv6_server/a10_cgnv6_server.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_server/a10_cgnv6_server.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        protocol: "tcp"
+        port_number: 8080
+        sampling_enable:
+        - counters1: "all"
```
Comment (hthompson6) [inline examples/a10_cgnv6_server/a10_cgnv6_server.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_server/a10_cgnv6_server.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        action: "enable"
+      sampling_enable: 
+      - counters1: "all"
```
Comment (hthompson6) [inline examples/a10_cgnv6_server/a10_cgnv6_server.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_service_group/a10_cgnv6_service_group.yml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+      state: "present"
+      get_type: "single"
+      shared_partition: "MyPartition"
+      protocol: "tcp"
+      name: "MyName"
+      user_tag: "usrtg1"
+      sampling_enable:
+      - counters1: "all"
+      member_list: 
+      - port: 8080
```
Comment (hthompson6) [inline examples/a10_cgnv6_service_group/a10_cgnv6_service_group.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_service_group/a10_cgnv6_service_group.yml [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      state: "present"
+      get_type: "single"
+      shared_partition: "MyPartition"
+      protocol: "tcp"
+      name: "MyName"
+      user_tag: "usrtg1"
+      sampling_enable:
+      - counters1: "all"
```
Comment (hthompson6) [inline examples/a10_cgnv6_service_group/a10_cgnv6_service_group.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_service_group/a10_cgnv6_service_group.yml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+      member_list: 
+      - port: 8080
+        sampling_enable:
+        - counters1: "all"
```
Comment (hthompson6) [inline examples/a10_cgnv6_service_group/a10_cgnv6_service_group.yml]: should indented by 2 spaces

File: examples/a10_cgnv6_service_group/a10_cgnv6_service_group_member.yml [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      state: "present"
+      partition: "shared"
+      get_type: "single"
+      port: 8080
+      sampling_enable:
+      - counters1: "all"
```
Comment (hthompson6) [inline examples/a10_cgnv6_service_group/a10_cgnv6_service_group_member.yml]: should indented by 2 spaces

**Comment by mmdurrant** [comment]
The lines mentioned regarding indents - following lines are siblings and don't require indentation.

**Pull request merged**